crates/genesis/src/lib.rs

```diff
@@ -217,6 +224,55 @@ pub struct GenesisAccount {
         deserialize_with = "deserialize_storage_map"
     )]
     pub storage: Option<HashMap<B256, B256>>,
+    /// The account's private key.
+    #[serde(
+        rename = "secretKey",
+        default,
+        skip_serializing_if = "Option::is_none",
+        with = "secret_key"
+    )]
+    pub private_key: Option<LocalWallet>,
+}
+
+/// serde support for `secretKey` in genesis
+pub mod secret_key {
+    use alloy_primitives::Bytes;
+    use alloy_signer::LocalWallet;
+    use k256::{ecdsa::SigningKey, SecretKey};
```

Comment:
this should not be required here,
the LocalWallet type should have all the necessary functions, if not we should add

Reply:
Not a great idea to serialize pk by default as raw bytes